stacker/providers/aws/default.py

```diff
@@ -72,6 +72,54 @@ def _throttling_checker(exc):
                               retry_checker=_throttling_checker)
 
 
+def s3_fallback(self, fqn, template_url, parameters, tags, create_or_update, **kwargs):
```

Comment:
It also looks like kwargs is only used for the change set name, so I'd just have an argument for that here - and default to None so you can get rid of kwargs (which always feels error prone)